### DISCUSSION TITLE
Attempt to authenticate with the salt master failed with timeout error

### DISCUSSION THREAD

**Discussion by utmostred** [Security-Onion-Solutions/securityonion#8674] [Unsupported Versions]
Trying to connect a sensor to a master server and can't seem to figure out what is the problem.

Can someone please point me at the right direction? Any help would be very much appreciated. :) 

Here is some sosetup.log from the sensor 

> `Loaded plugins: versionlock
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/docker-ce-stable/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/epel/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/extras/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/salt/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/securityonion/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/updates/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh4_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/wazuh_repo/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
repo id             repo name                                         status
base                CentOS-7 - Base                                   enabled: 0
centosplus          CentOS-7 - Plus                                   disabled
docker-ce-stable    Docker CE Stable - x86_64                         enabled: 0
epel                Extra Packages for Enterprise Linux 7 - x86_64    enabled: 0
extras              CentOS-7 - Extras                                 enabled: 0
saltstack           SaltStack repo for RHEL/CentOS 7 PY3              enabled: 0
securityonion       Security Onion Repo                               enabled: 0
updates             CentOS-7 - Updates                                enabled: 0
wazuh4_repo         Wazuh repository                                  enabled: 0
wazuh_repo          Wazuh repository                                  enabled: 0
repolist: 0
2022-09-05T07:42:33Z | I | Executing command: yum -v -y update centos-release
Not loading "fastestmirror" plugin, as it is disabled
Loading "versionlock" plugin
Config time: 0.014
Yum version: 3.4.3
rpmdb time: 0.000
Setting up Package Sacks
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.
http://repocache.securityonion.net/file/securityonion-repo/base/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to onion1:3142; Connection refused"
Trying other mirror.`

`Configuring minion type as sensor
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (1 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (2 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (3 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (4 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (5 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (6 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (7 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (8 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (9 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (10 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (11 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (12 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (13 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (14 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (15 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (16 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (17 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (18 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (19 / 20)...
Executing command with retry support: accept_salt_key_remote
Accept the key remotely on the manager
Results: The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys. (0)
Did not find expectedOutput: 'going to be accepted' in the output below from running the command: 'accept_salt_key_remote'
<Start of output>
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
The key glob 'so-sensor_sensor' does not match any unaccepted keys.
<End of output>
Command failed with exit code 0; will retry in 10 seconds (20 / 20)...
Command continues to fail; giving up.`
![image](https://user-images.githubusercontent.com/111489994/188820598-8b2cde83-114c-4f6f-a226-8cd932946c67.png)

**Comment by reyesj2** [comment]
What's between your forward node & manager node? Any firewall? Verify that you have end to end connectivity first. 

Also ensure your manager node is running.
When you run (on forward node 'sensor')
```
sudo ./so-setup iso 
```
After selecting 'Distributed' -> 'Existing Deployment' 
Make sure you're typing in the manager hostname correctly (case sensitive) along with verifying your password for the 'soremote' account on the manager node

**Reply by utmostred** [reply]
Don't know what firewall to enable. I can ping the Manager node from the Sensor node that I'm trying to connect but I can't ping the Sensor node from the Manager node. Any pointers? I'm new to Security Onion.

**Reply by utmostred** [reply]
and the Manager node is running when I'm trying to run the sensor.

**Comment by dougburks** [comment]
Have you verified that your nodes can communicate using the ports shown at https://docs.securityonion.net/en/2.3/firewall.html#node-communication?
